### DISCUSSION TITLE
configure fuzzing/symbolic balance using CLI

### DISCUSSION THREAD

**Discussion by ghost** [UnitTestBot/UTBotJava#2735] [Q&A] [answered]
The Settings page for UnitTestBot's IntelliJ IDEA plugin offers a "Test generation method" slider, with "Fuzzing 100% / 0% Symbolic execution" at one extreme and "Fuzzing 0% / 100% Symbolic execution" at the other. I cannot find any similar configuration knob among the UnitTestBot CLI's command-line flags.

**Is it possible to configure this fuzzing/symbolic balance using the CLI?** If so, how? If not, then what default balance is used, and where is that configured in the code? Thanks for any pointers!

**Comment by tyuldashev** [comment, marked as answer]
Hi,
At the moment it cannot be set in CLI, default ratio is 0.1 for fuzzing and 0.9 for symbolic.

To get details about the code you may look into this unfinished pull request, which introduces such option - https://github.com/UnitTestBot/UTBotJava/pull/2496

**Comment by ghost** [comment]
Perfect! Thanks for the info and related-pull-request pointer, Tumur.
